src/components/DivElement.js

```diff
@@ -1,6 +1,13 @@
 const HTMLElement = require('./HTMLElement');
 
 // Define class here
+class DivElement extends HTMLElement {
+  constructor(content) {
+    super();
```

Comment:
I struggled a bit with super() as well, but without passing the `content` argument in the super call, I don't think you are in fact leveraging the HTMLElement class. The method that finally worked for me was the following: 

```javascript
class DivElement extends HTMLElement {
  constructor(content) {
    super('div', content);
  }
}
```
this allows you to leverage the entire HTMLElement class by predefining the tag as `div` and passing the content along to the parent class.

Reply:
Thanks for pointing that out - appreciate it!